thirdeye/thirdeye-frontend/app/pods/components/rootcause-heatmap/component.js

```diff
@@ -41,10 +47,16 @@ export default Component.extend({
 
   breakdowns: null, // {}
 
-  selectedUrn: null, // Set
+  selectedUrn: null, // ""
```

Comment:
Can we remove all these comments? Do we need them?

Reply:
removed